include/rangers-api/Application/Player/StateParameter.h

```diff
@@ -0,0 +1,9 @@
+#pragma once
+
+namespace app::player {
+    class StateParameter {
```

Comment:
Starting braces should be started on a line to keep consistent with the original repository.